pubsub/api/PubsubTest/PubsubTest.cs

```diff
@@ -42,15 +42,60 @@ public class PubsubTest : IDisposable
             Command = "Pubsub"
         };
 
+        /// <summary>
+        /// Handle a special race condition that can occur when deleting
+        /// something:
+        /// 1. Delete request times out.
+        /// 2. Delete operation continues on server and succeeds.
+        /// 3. Later requests to delete the same entity see NotFound error.
+        /// </summary>
+        /// <param name="delete">The delete operation to run.</param>
+        /// <returns>An action to run inside Eventually().</returns>
+        Action HandleDeleteRace(Action delete)

```

Comment:
Normally, I'd test that this delegate is not null, however not sure if you should do it in sample code...

Reply:
If someone passes in a null delegate, the test fails, we see an error report and fix it.  Which is exactly the same as what we'd have to do if we added a null test here.